main.css

```diff
@@ -26,4 +26,11 @@ body {
   background: #798EF6;
   border-radius: 7px;
   margin: 5px;
+  transform: scale(1);
+  transform-style: preserve-3d;

```

Comment:
I'm not sure we need this right now. It's for card flip animation

Reply:
If you tell about this, you are right
```css
transform-style: preserve-3d;
```